backend/frontend/templates/frontend/users_and_teams_nav/page.html

```diff
@@ -3,9 +3,13 @@
 {% block content %}
     <main>
         <div class="mx-auto max-w-7xl py-6 sm:px-6 lg:px-8">
-
-            <h1 class="{{ styles.heading1 }}">Teams</h1>
-            
+            <div class="flex gap-2 flex-col">
+                <h1 class="{{ styles.heading1 }} pb-2">Teams</h1>
+                <form method="post" action="" class="mb-3 flex items-center gap-1">
```

Comment:
This is what the different HTTP verbs mean:

GET: fetching and filtering
POST: Creating or updating
PUT: updating
DELETE: killing with fire
OPTIONS: header check

POST doesn't seem like the most appropriate thing here. We should always try to use things for their intended purpose.